### ISSUE TITLE
The storage quota and eviction docs need a specific PWA section

### ISSUE THREAD

**Issue by captainbrosset** [mdn/content#25547]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria

### What information was incorrect, unhelpful, or incomplete?

Thanks to @szewai, who works on WebKit, I now know that WebKit is in the process of working out on new storage implementation details for home-screen installed web apps. Different quotas and eviction criteria might get applied for installed PWAs.

This means that [the MDN storage quota/eviction docs](https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria) should get updated with PWA-specific information, when we have it.

My understanding is that this is WebKit specific and that Chromium and Gecko keep the same process for web apps in the browser, and installed PWAs.

### What did you expect to see?

A new section specific to PWA for Safari/WebKit.

**Comment by Josh-Cena** [comment]
@captainbrosset Are you planning to work on this, or do you have any references so someone else could work on it?

**Comment by captainbrosset** [comment]
No, unfortunately this isn't on my radar at the moment. I suggest asking @szewai to check if they can provide information about the above.

**Comment by wbamberg** [comment]
This isn't actionable for us at the moment - we don't know which changes are being requested - so I'm going to close it. Happy to take care of it when @szewai or another expert has details to share.
